### PR TITLE
dApp Staking v3

### PR DESCRIPTION
**Pull request by Dinonard**
dApp Staking v3 documentation.

### CODE REVIEW

File: docs/learn/dapp-staking.md [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+Periods are another _time unit_ in dApp staking. They are expected to be more lengthy than eras.
+Each period is denoted by a number, e.g. **1**, which increments each time a new period begins.
+
+Each period consists of two subperiods:
+* `Voting`
+* `Build&Earn`
+
+Period beginning is marked by the `Voting` subperiod, after which follows the `Build&Earn` subperiod.
+
+Stakes are **only** valid throughout a period. When new period starts, all stakes are reset to **zero**. This helps prevent projects remaining staked due to inertia of stakers, and makes for a more dynamic staking system. Staker doesn't need to do anything for this to happen, it is automatic.
```
Comment (andabak) [inline docs/learn/dapp-staking.md:36]: This helps prevent projects remaining staked due to inertia of stakers, and makes for a more dynamic staking system.
-->
Protocol dynamic prevents inactive projects having high stakes due to inertia of stakers while at the same time allows more visibility for dApps joining the protocol and equal chance in attracting stakers' attention.

File: docs/learn/dapp-staking.md [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+Neither stakers nor dApps earn rewards during `Voting` subperiod - no new rewards are generated after `Voting` subperiod ends.
+However, as mentioned before, it's crucial for dApps to use this time to market themselves and attract stakers, and for stakers
+to asses which projects they want to support.
+
+It is important to award projects which the staker believes will bring value to the network and possibly themselves.
+Voting on a poor project means that poor performance is rewarded, damaging the network instead of improving it.
+
+#### Build&Earn
+
+`Build&Earn` subperiod consists of one or more eras, therefore its length is expressed in eras.
```
Comment (andabak) [inline docs/learn/dapp-staking.md:62]: would delete this (no need): "therefore its length is expressed in eras."

**Review by andabak — APPROVED** [review]

File: docs/learn/dapp-staking.md [hunk 3]
```diff
@@ -0,0 +1,319 @@
+---
+sidebar_position: 11
+title: dApp Staking
+toc_max_heading_level: 3
+---
+
+## Introduction
```
[the remark this answers is not in the document]
Comment (Dinonard) [reply docs/learn/dapp-staking.md:7]: In general, I don't have anything against the idea. I believe you'll know this better than me.

For the 2nd bullet though, personally I would prefer to keep the detailed overview of all features in the single place.
Extracting the info from the main _dapp-staking.md_ file into other places, and making it more succinct sounds good.

File: docs/learn/dapp-staking.md [hunk 4]
```diff
@@ -0,0 +1,319 @@
...
+The following chapters will provide an overview of the functionality and terminology with accompanying examples.
+
+## Functionality Overview
+
+### Eras
+
+Eras are the basic _time unit_ in dApp staking and their length is measured in the number of blocks.
+
+Eras are not expected to last long, e.g. current production networks era length is roughly 1 day (7200 blocks).
+After an era ends, it's usually possible to claim rewards for it, if user or dApp are eligible.
```
[the remark this answers is not in the document]
Comment (Dinonard) [reply docs/learn/dapp-staking.md:23]: Sure, let me rename it.

File: docs/learn/dapp-staking.md (hunk 1 above)
Comment (Dinonard) [reply docs/learn/dapp-staking.md:36]: Applied.

File: docs/learn/dapp-staking.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Dinonard) [reply docs/learn/dapp-staking.md:7]: Another part we'll probably need to add later is about params of the network(s).

[the remark this answers is not in the document]
Comment (andabak) [reply docs/learn/dapp-staking.md:7]: OK. Then it goes out as it, and I do a follow up PR with some reorganization of dAppStaking content in general (
- without breaking this master doc but perhaps moving it somewhere "deeper" in the docs
- adjusting the first time reader info with succinct info

[the remark this answers is not in the document]
Comment (andabak) [reply docs/learn/dapp-staking.md:7]: https://github.com/AstarNetwork/astar-docs/issues/556

**Pull request merged**